### DISCUSSION TITLE
How can we see the time course of  CYP3A?

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#993] [Questions & Problems] [answered]
Hello team,

I wonder how can I get such a figure?
![image](https://user-images.githubusercontent.com/92715703/137692428-662b42a9-9061-4510-8d88-8bbb8b8bce51.png)

It will be great if you could give me some feedback. Thank you very much.

Have a nice day.

Best,
Carrie.

**Comment by StephanSchaller** [comment, marked as answer]
Hi Carrie,

sure, if you (just ignore the compound name):
1. model the depletion of CYP3A4 by drug interactions (right-click on *inhibition*) 
![image](https://user-images.githubusercontent.com/25477643/137711659-efebb834-2ef1-4fe1-81b3-3aafe3cd6830.png)
![image](https://user-images.githubusercontent.com/25477643/137712144-f9271aa4-16c6-4af6-be98-a5fd248128f0.png)

and then 
2. create the simulation (with a multiple-dose scenario for the perpetrator) and activate the interaction
![image](https://user-images.githubusercontent.com/25477643/137712293-1c566acd-ef3e-4fc0-9715-c8236ee130f4.png)

you only need to 
3. select the CYP3A4 concentrations (in liver and gut segment) for plotting when running the simulation

**Comment by Carrie-nothing** [comment]
Dear Stephan,

What you have shown me is the enzyme activity curve under irreversible inhibition. 

May I ask if the drug I need to do is reversible inhibition of CYP3A4 then the step 1 should also be adopted?

**Comment by wang000115** [comment]
Hello everyone, I have encountered a difficulty. When we simulated, we chose CYP3A4 concentration, but we did not get% remaining CYP3A activity. How should I choose to get a percentage? If I could receive a reply, I would be very grateful.
